docs/pom.xml

```diff
@@ -110,17 +66,17 @@
 					<plugin>
 						<groupId>org.asciidoctor</groupId>
 						<artifactId>asciidoctor-maven-plugin</artifactId>
-						<configuration>
-							<attributes>
-								<project-version>${project.version}</project-version>
```

Comment:
Why are you removing this?

Reply:
This should already be present: https://github.com/spring-cloud/spring-cloud-build/blob/master/pom.xml#L1149